### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by brianteeman**
4.1-dev has been made the default branch - but no one updated the readme :)

### CODE REVIEW

**Comment by zero-24** [comment]
Didnt we also have a contribution.md in the .github folder that should be updated with the versions too?

**Comment by zero-24** [comment]
Do you want to do it here or should there be a different PR for that?

**Comment by brianteeman** [comment]
Done as requested

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 9cf4cb22eb28477dc90d305569812b6ac5d7eed6<br /><br />By review and checked that the build status badges are shown right on https://github.com/joomla/joomla-cms/blob/9cf4cb22eb28477dc90d305569812b6ac5d7eed6/README.md .<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36811">issues.joomla.org/tracker/joomla-cms/36811</a>.</sub>

**Comment by Fedik** [comment]
I want to point, that the readme a bit misleading. It not related to current PR, but in general.

After 4.1 will be released:
PR for bugfixes 4.1-dev,
If you want suggest new feature then 4.2-dev.
And the same for future version shift.

But I guess this need to be discussed somewhere else :)

File: README.md [hunk 1]
```diff
@@ -45,9 +45,9 @@ git clone git@github.com:joomla/joomla-cms.git
 ```bash
 cd joomla-cms
 ```
-- Go to the 4.0-dev branch:
+- Go to the 4.1-dev branch:
 ```bash
-git checkout 4.0-dev
+git checkout 4.1-dev
```
Comment (rjharishabh) [inline README.md:50]: `4.1-dev` is the default branch so I think no need to checkout it

Comment (richard67) [reply README.md:50]: No need, but also no harm when doing it, and it makes it easier to verify this file with the file from the 4.2-dev branch where a checkout is necessary.

Comment (rjharishabh) [reply README.md:50]: Agree

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 9cf4cb22eb28477dc90d305569812b6ac5d7eed6<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36811">issues.joomla.org/tracker/joomla-cms/36811</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36811">issues.joomla.org/tracker/joomla-cms/36811</a>.</sub>

**Comment by bembelimen** [comment]
Thx, had it on my list, but hadn't the time yet, saves me some time :)

**Comment by brianteeman** [comment]
teamwork ;)

**Pull request merged**